src/module-1/card/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product</title>
+    <link rel="stylesheet" href="../styles/main.css" />
+    <link rel="stylesheet" href="../styles/card.css" />
+  </head>
+  <body>
+    <div class="product-wrapper">
+      <div class="product">
+        <div class="product__image">
+          <img
+            src="../images/products/iphone-x.png"
+            alt="Apple iPhone X 128GB"
+          />
+        </div>
+        <div class="product__rating-price">
+          <div class="product__rating"><span>3.4</span></div>
+          <span class="product__price">$899</span>
+        </div>
+        <div class="product__ifno">
+          <h3 class="product__name">Apple iPhone X 128GB</h3>
+          <span class="product__description">
+            Redesigned from scratch and completely revised.
+          </span>
+        </div>
+        <div class="product__buttons">
+          <button class="button button__secondary">
```

Comment:
Modifier name starts with one `_`: `button_secondary`
https://en.bem.info/methodology/naming-convention/#block-modifier-name

I see you use correct BEM-naming the module-3 homework